lib/inflate.js

```diff
@@ -241,7 +241,11 @@ Inflate.prototype.push = function (data, mode) {
     }
 
     if (strm.next_out) {
-      if (strm.avail_out === 0 || status === c.Z_STREAM_END || (strm.avail_in === 0 && (_mode === c.Z_FINISH || _mode === c.Z_SYNC_FLUSH))) {
+      if (
```

Comment:
Please roll back this change. That's not related to subject of commit. It's better to not touch coding style in this PR.

Reply:
ok, i reverted that style change, but there is a change to those conditions, and having them all on one line makes it hard to read

Reply:
Sorry, i did not found difference, because change was too big and looked like style update. I think, it will be better to recommit history later, in the end.

Could you explain what exactly was changed and why?